src/blueprints/ember-addon/package.json

```diff
@@ -30,10 +30,6 @@
   "repository": "",
   "license": "MIT",
   "author": "",
-  "workspaces": [
-    "<%= options.locations.addon %>",
-    "<%= options.locations.testApp %>"
-  ],
```

Comment:
For `pnpm` projects, the workspace locations are defined in [`pnpm-workspace.yaml`](https://github.com/ijlee2/ember-codemod-v1-to-v2/blob/0.3.2/src/blueprints/ember-addon/pnpm-workspace.yaml).